addons/website/tests/test_assets.py

```diff
@@ -12,9 +12,10 @@ class TestWebsiteAssets(odoo.tests.HttpCase):
     def test_01_multi_domain_assets_generation(self):
         Website = self.env['website']
         Attachment = self.env['ir.attachment']
+        Website.create({'name': 'Second Website'})
```

Comment:
I am fine keeping it but I'd just comment it please otherwise it's hard to figure what is this line purpose, but as you prefer really, thanks for the fix :)